[ERF] Port CW2 Main Bodys to new CW2 motor plate (as of July 2023)

EmJay276 opened the pull request:
This PR applies changes done to the CW2 motor plate in July 2023 to all Clockwork2 main bodys.
![292590340-58c7e930-77d1-4c09-8238-f49af9075681](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/8619525/96df3f97-4458-4cc4-aada-05f38f7a0375)

Additonally adds a M6 bowden fitting version of the main body without sensor and a stronger latch.

Tyler8299 commented:
most all mid plates will need updated too as most are the older thicker design

EmJay276 commented:
> most all mid plates will need updated too as most are the older thicker design single sensor and dual sensor with ecas

What thickness did change? 
I didn't see any differences in the CAD data despite the red "edge" that was removed/shrinked.

I am already using the new M6 body without any issues [SB_CW2_Body_M6](https://github.com/EmJay276/ERCF_v2/blob/CW2_new_motor_plate/Recommended_Options/ERF_Filament_Cutter/Stls/4_No_Sensors/SB_CW2_Body_M6.stl)

Tyler8299 commented:
![Screenshot 2023-12-23 124726](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/121695166/5a098a81-e45f-43f5-b60e-8c3d7d9592e2)
![Screenshot 2023-12-23 133750](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/121695166/929771e5-a6cc-43db-bf0e-9b1316ed9bc8)
![Screenshot 2023-12-23 133809](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/121695166/5c43894c-d2a3-4178-bed6-9ac5ed12a82f)
![Screenshot 2023-12-23 133849](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/121695166/20f13f68-5d98-48be-9049-18558ab0b871)

Tyler8299 commented:
all the mid plates we offer have the thicker tongue

Tyler8299 commented:
![Screenshot 2023-12-23 134252](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/121695166/cd3750b8-6883-49a2-ac49-f1a53b39ade0)
![Screenshot 2023-12-23 134301](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/121695166/cb53ca84-8027-4c28-a7f3-b284dbe21ea6)

Tyler8299 commented:
and the latch and motor plate is universal between what we offer for no confusion

EmJay276 commented:
My designs are all based on the Main Bodys of the Filametrix Repo with the sensors.
I just removed the additional material that was marked in red in the first post.
If you didn't change the Filametrix parts, they will fit. My parts are already approved by @sorted01 in the original Filametrix repo. https://github.com/sorted01/Filametrix/pull/40
The original [latch](https://github.com/Enraged-Rabbit-Community/ERCF_v2/blob/master/Recommended_Options/ERF_Filament_Cutter/Stls/%5Ba%5D_SB_Latch_ECAS.stl) is compatible with the additonal M6 main body, I just added a little more sturdy handly if you use the M6 body.

![grafik](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/8619525/40c025d3-9c67-4d26-8ff1-fa48a1af2190)

![grafik](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/8619525/af06ede3-2f44-47c2-a1c1-3b43b55463b2)

EmJay276 commented:
I also converted the [Toolhead_Modifications](https://github.com/Enraged-Rabbit-Community/ERCF_v2/tree/master/Recommended_Options/Toolhead_Modifications) to the new motor plate for consistency, see https://github.com/Enraged-Rabbit-Community/ERCF_v2/pull/7/commits/0afea0ec3d1715478a2f8130dec03c5196756cab

Anollya commented:
FYI:
Bigtreetech has yesterday updated the EBB SB2240_2209 CAN STLs including the main_body and the cable cover -> V1.2
https://github.com/bigtreetech/EBB/tree/master/EBB%20SB2240_2209%20CAN
Is it interesting for the Filament Cutter Mod?

EmJay276 commented:
> FYI: Bigtreetech has yesterday updated the EBB SB2240_2209 CAN STLs including the main_body and the cable cover -> V1.2 https://github.com/bigtreetech/EBB/tree/master/EBB%20SB2240_2209%20CAN Is it interesting for the Filament Cutter Mod?
> 

If I understand this correct, the new EBB SB2240_2209 CAN STLs is for the new CW2 motor plate, isn't it?

Anollya commented:
Yes, for the new motor plate and it has for the new EBB cover a rounded edge.
I have made a new motor plate cut together with prusa slicer -> new btt motor plate with ecas und toolhead sensor.
I think this is the first one.
 
![2024-01-02 16_07_04-_Unbenannt - PrusaSlicer-2 7 1 basiert auf Slic3r](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/10775115/c5c7a378-bccc-4763-95a9-3b1773af50fd)

May be anyone would test it:
[EBB CAN  Motorplate ECAS Toolheatsensor.zip](https://github.com/Enraged-Rabbit-Community/ERCF_v2/files/13811238/EBB.CAN.Motorplate.ECAS.Toolheatsensor.zip)

jfgordon2 commented:
I’m just some dude trying to print your files for his build, but I’m wondering if you know why the ECAS latch has to get thinner closer to its hinge, or if we can keep the thickness from the regular latch and only make room for the ECAS for the modification?

![IMG_7329](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/55799997/b08d14e1-a3e8-4f77-808a-340e98541cdd)

EmJay276 commented:
> I’m just some dude trying to print your files for his build, but I’m wondering if you know why the ECAS latch has to get thinner closer to its hinge, or if we can keep the thickness from the regular latch and only make room for the ECAS for the modification?
> 
> 

I can have a look at it on the weekend, we can probably keep the regular thickness.

Would you be available for testing a new ECAS latch if I design it?

jfgordon2 commented:
> > I’m just some dude trying to print your files for his build, but I’m wondering if you know why the ECAS latch has to get thinner closer to its hinge, or if we can keep the thickness from the regular latch and only make room for the ECAS for the modification?
> 
> I can have a look at it on the weekend, we can probably keep the regular thickness.
> 
> Would you be available for testing a new ECAS latch if I design it?

Of course!

jfgordon2 commented:
> Yes, for the new motor plate and it has for the new EBB cover a rounded edge.
> 
> I have made a new motor plate cut together with prusa slicer -> new btt motor plate with ecas, toolhead sensor and Filamentrix Cutter.
> 
> I think this is the first one.
> 
>  
> 
> ![2024-01-02 16_07_04-_Unbenannt - PrusaSlicer-2 7 1 basiert auf Slic3r](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/10775115/c5c7a378-bccc-4763-95a9-3b1773af50fd)
> 
> 
> 
> May be anyone would test it:
> 
> [SB_CW2_new_motor_plate_BTT_EBB1_2_Toolheadsenor_ECAS_Filamentrix_Cutter.zip](https://github.com/Enraged-Rabbit-Community/ERCF_v2/files/13811316/SB_CW2_new_motor_plate_BTT_EBB1_2_Toolheadsenor_ECAS_Filamentrix_Cutter.zip)
> 
> 
> 
> 
> 
> 

@Anollya I printed this off and it seems dimensionally accurate, but the tongue snapped off with the eSUN ABS+ I used, so I'm reordering something with better layer adhesion to fully assemble the stealthburner with this and the EBB cable cover.

Tyler8299 commented:
Reading this makes another +1 in my book for the older style with beefier tongue.

EmJay276 commented:
> Reading this makes another +1 in my book for the older style with beefier tongue.

The beefier tongue is the newer style

EmJay276 commented:
@jfgordon2 
I just added the stronger [ECAS latch](Recommended_Options/ERF_Filament_Cutter/Stls/[a]_SB_Latch_ECAS.stl)
In CAD it fits, would be a charm if you could test it.
![grafik](https://github.com/Enraged-Rabbit-Community/ERCF_v2/assets/8619525/4c48862d-def7-4e22-9838-c2ecdfa1df9f)

ndanyluk commented:
Hi @EmJay276! Just wanted to say thanks for doing this! I have a 1.0 release CW2 and didn't want to roll back to the older style.

Quick question for you: does this PR use the older 7mm ball bearings or the 5.5mm ones per #30?

EmJay276 commented:
@ndanyluk thank you very much! 
Unfortunately I don't know which size of the ball bearing I used, I'm currently on vacation and don't have access to the files.
If I should remodel the parts using another ball bearing size, please let me know.
I will do it as soon as I am back, but this will take another 4 weeks.
